django/db/backends/base/creation.py

```diff
@@ -268,6 +272,24 @@ def destroy_test_db(self, old_database_name=None, verbosity=1, keepdb=False, num
             settings.DATABASES[self.connection.alias]["NAME"] = old_database_name
             self.connection.settings_dict["NAME"] = old_database_name
 
+    def _restore_initial_data_migrations(self):
+        # Load initial data migrations, if any.
+        if hasattr(connections[self.connection.alias], "_test_serialized_contents"):
+
+            # Don't import django.core.management if it isn't needed.
+            from django.core.management import call_command
```

Comment:
That's what I saw in `create_test_db` function, so I guess there were some issues with this file imports? I just replicated this local import logic.